src/commands/purge.test.js

```diff
@@ -0,0 +1,101 @@
+const should = require(`should`);
+const sinon = require(`sinon`);
+require(`should-sinon`);
+
+const util = require(`./../util`);
+const testData = require(`./../test.data`)
+const testSubject = require(`./purge`);
+
+const testUrl = `https://example.com`;
+
+// Scope vars that we’ll redefine on each test.
+let options, purgeStub, softPurgeStub, Fastly;
+
+// Begin testing.
+describe(`purge`, () => {
+  // Before and after each test, we reset some stuff
+  // to ensure a fresh set of arguments. See .beforeEach()
+  // and .afterEach() near the bottom of this file.
+
+  it(`should stop without an API key`, (done) => {
+    delete options.apikey;
+    testSubject(options, Fastly, util);
+    util.apiKeyPresent.should.be.calledOnce();
+    Fastly.should.not.be.called();
+    done();
+  });
+
+  it(`should stop without a Service ID`, (done) => {
+    delete options.serviceid;
+    testSubject(options, Fastly, util);
+    util.serviceIdPresent.should.be.calledOnce();
+    Fastly.should.not.be.called();
+    done();
+  });
+
+  it(`should instantiate Fastly with API key`, (done) => {
+    testSubject(options, Fastly, util);
+    Fastly.should.be.calledOnce();
+    Fastly.getCalls()[0].args.should.deepEqual([options.apikey]);
+    done();
+  });
+
+  it(`should instantiate ResponseHandler with message`, (done) => {
+    testSubject(options, Fastly, util, testUrl);
+    util.ResponseHandler.should.be.calledOnce();
+    util.ResponseHandler.getCalls()[0].args[testData.argPositionOptions].should.equal(`Purged URL: ${testUrl}`);
```

Comment:
Since `ResponseHandler` takes only one argument, I propose we do here the same thing we do on line `39` on this file:
```
util.ResponseHandler.getCalls()[0].args.should.deepEqual([`Purged URL: ${testUrl}`]);
```